### ISSUE TITLE
Reimplement `NetJsonEdgeSerializer` and `NetJsonNodeSerializer`

### ISSUE THREAD

**Issue by b1rger** [acdh-oeaw/apis-core-rdf#847]
talk to frontend what exactly they need for the visualisation

**Comment by b1rger** [comment]
@sennierer @stefanprobst IIRC you discussed this, maybe you can post your findings here so someone can implement this

**Comment by stefanprobst** [comment]
i think we agreed to return a list of edges:

```json
{
  "count": 100
  "limit": 10,
  "offset": 0,
  "edges": [
    { 
      "id": "edge-1",
      "source": { "id": "node-1", "label": "The first node", "type": "person" },
      "target": { "id": "node-2", "label": "The second node", "type": "person" },
      "type": { "id": "relation-type-1", "label": "is related to" },
      "properties": {}
    },
    // ...
  ]
}
```

@sennierer we have not talked about `properties` but this could be useful for start/end date or similar which may not exist in all datasets

**Comment by b1rger** [comment]
Oke, so the default, once #1124 is merged, would be this:

```json
{
  "count": 11,
  "next": "http://localhost:8000/api/relations.relation/?format=json&limit=5&offset=5",
  "previous": null,
  "results": [
    {
      "url": "http://localhost:8000/api/sample_project.issiblingof/1/?format=json",
      "label": "Mustrum Ridcully is sibling of Hughnon Ridcully",
      "content_type_key": "sample_project.issiblingof",
      "content_type_name": "is sibling of",
      "subj": {
        "url": "http://localhost:8000/api/sample_project.person/2/?format=json",
        "label": "Mustrum Ridcully",
        "content_type_key": "sample_project.person",
        "content_type_name": "person"
      },
      "obj": {
        "url": "http://localhost:8000/api/sample_project.person/3/?format=json",
        "label": "Hughnon Ridcully",
        "content_type_key": "sample_project.person",
        "content_type_name": "person"
      }
    },
    {
      "url": "http://localhost:8000/api/sample_project.iscousinof/2/?format=json",
      "label": "Galder Weatherwax is cousin of Esmerelda Weatherwax",
      "content_type_key": "sample_project.iscousinof",
      "content_type_name": "is cousin of",
      "subj": {
        "url": "http://localhost:8000/api/sample_project.person/6/?format=json",
        "label": "Galder Weatherwax",
        "content_type_key": "sample_project.person",
        "content_type_name": "person"
      },
      "obj": {
        "url": "http://localhost:8000/api/sample_project.person/9/?format=json",
        "label": "Esmerelda Weatherwax",
        "content_type_key": "sample_project.person",
        "content_type_name": "person"
      }
    },
    // ...
  ]
}
```
which is very similar

**Comment by b1rger** [comment]
#1124 has no been merged an released in v0.25.0, so I'm closing this
